version/version.go

```diff
@@ -10,3 +10,11 @@ var (
 	// Revision is filled with the VCS revision. Filled in at linking time.
 	Revision = ""
 )
+
+func GetVersion() string {
+	return Version
+}
+
+func GetRevision() string {
+	return Revision
+}
```

Comment:
What @binbin-li as mentioned previously is correct. We don't need a `GetVersion()` for `Version` as it is already exposed.

What is discussed is something like
```go
func FullVersion() string {
    if Revision == "" {
        return Version
    }
    return Version + "+" + Revision
}
```

Reply:
Added FullVersion() and removed the getter and setter.